tests/integration/iop-remote-integration-test-defaults.yaml

```diff
@@ -0,0 +1,32 @@
+# This file provides some defaults for integration testing.
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  meshConfig:
+    accessLogFile: "/dev/stdout"
+    defaultConfig:
+      proxyMetadata:
+        ISTIO_META_DNS_CAPTURE: "true"
```

Comment:
do we want to test this set as true?  We are not enabling this by default in our MC docs.  Could you double check?

Reply:
This was set as true in #26995, I think it's good to test with it on so we can work on stabilizing it.

Reply:
i see thanks - okay to have this to true for master.  if this is cherry-picked to 1.8, I think it would be good to test what we want users to do.